Devscord.DiscordFramework/Services/EmbedMessagesService.cs

```diff
@@ -17,6 +19,17 @@ public Embed Generate(string title, string description, IEnumerable<KeyValuePair
             return builder.Build();
         }
 
+        public Embed Generate(string title, string description, Dictionary<string, Dictionary<string, string>> values)
+        {
+            var flatValues = new Dictionary<string, string>();
+            foreach (var value in values)
+            {
+                var valuesString = value.Value.Aggregate(String.Empty, (a, b) => (b.Key +" "+ b.Value));
```

Comment:
string możesz pisać z małej litery
+jak nie musisz, to nie dodawaj dodatkowych nawiasów